### PR TITLE
Unify HTCondor secondary with HTCondor and cleanup HTCondor secondary configs

### PR DESCRIPTION
**Pull request by sanjaysrikakulam**
This PR aims to clean up HTCondor secondary deployment and merge the HTCondor secondary with HTCondor.

Please review this PR carefully.

### CODE REVIEW

File: host_vars/sn06.galaxyproject.eu.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 htcondor_network_interface: ens802f0.223
 
-# Settings specific to the `usegalaxy_eu.htcondor` role.
-condor_daemons:
-  - COLLECTOR
-  - NEGOTIATOR
-  - MASTER
-  - SCHEDD
+# 15/03/2024: On sn06 HTCondor conf was manually adjusted to use port 9618 since the
```
Comment (bgruening) [inline host_vars/sn06.galaxyproject.eu.yml:4]: We can stop all job submissions on the Galaxy side via the Admin panel. So no official downtime is needed? Or we could simply stop all job-handlers?

Comment (sanjaysrikakulam) [reply host_vars/sn06.galaxyproject.eu.yml:4]: We can stop the handlers and restart the htcondor systemd service.

File: group_vars/htcondor/vars.yml [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Configure nodes in the HTCondor cluster.
 ---
-htcondor_server: "condor-cm.galaxyproject.eu"
```
Comment (mira-miracoli) [inline group_vars/htcondor/vars.yml:3]: should we update it also here then, if we want to name it by `build`?
https://github.com/usegalaxy-eu/vgcn-infrastructure/blob/e2cb8a4e5fb74bbcf210d175d15f40f321928a94/userdata.yaml.j2#L7

Comment (sanjaysrikakulam) [reply group_vars/htcondor/vars.yml:3]: In your [PR](https://github.com/usegalaxy-eu/vgcn-infrastructure/pull/283/files#diff-18205f92094934ee2fc32afb35cfc40957322c9f197d3707a753bc02484bd5ba) you have already removed it, right?

Comment (mira-miracoli) [reply group_vars/htcondor/vars.yml:3]: yes, it is "baked-in" here:
https://github.com/usegalaxy-eu/vgcn/blob/2a80ce37ca3f1179c16bdfc026e7141dfb71fe9e/ansible/group_vars/condor.yml#L2

**Review by mira-miracoli — APPROVED** [review]
thank you!

File: group_vars/htcondor/vars.yml (hunk 2 above)
Comment (sanjaysrikakulam) [reply group_vars/htcondor/vars.yml:3]: Yup, it's baked into the image itself.

**Pull request merged**